compiler/src/iree/compiler/Codegen/Common/GPU/GPUNestedLayoutDistributionPatterns.cpp

```diff
@@ -563,6 +563,223 @@ struct DistributeMultiReduction final
   int64_t maxBitsPerShuffle;
 };
 
+/// The lowering for Contract is performed in three steps (similar to above
+/// multi_reduction):
+///   1. Local Contract: Each thread performs operations on its locally
+///   distributed elements.
+///   2. Thread Reduction: Threads collectively reduce the results from step 1
+///   across threads,
+///      using a butterfly shuffle if distribution occurs along the reduction
+///      dimension.
+///   3. Accumulator Reduction: Each thread combines its intermediate results
+///   with its held accumulator.
+///
+/// Currently, reduction across multiple warps is not supported.
```

Comment:
let's try to stick to the portable naming like in the gpu dialect

```suggestion
/// Currently, reduction across multiple subgroups is not supported.
```